frontend/packages/dev-console/src/components/topology/operators/TopologyOperatorBackedPanel.tsx

```diff
@@ -30,13 +49,8 @@ const TopologyOperatorBackedPanel: React.FC<TopologyOperatorBackedPanelProps> =
       <div className="overview__sidebar-pane-head resource-overview__heading">
         <h1 className="co-m-pane__heading">
           <div className="co-m-pane__name co-resource-item">
-            <ResourceIcon kind={data.operatorKind || 'Operator'} />
-            <ResourceLink
-              kind={referenceFor(resource)}
-              name={name}
-              namespace={resource.metadata.namespace}
-              hideIcon
-            />
+            <ResourceIcon kind={referenceFor(resource)} />
+            <Link to={getOperatorLink(resource)}>{name}</Link>
```

Comment:
You can reuse `ManagedByOperatorResourceLink` here instead of custom implementation.

Reply:
Updated.